js/parts-3d/Column.js

```diff
@@ -265,6 +259,7 @@ addEvent(Series, 'afterInit', function () {
                 z = (stacks.totalStacks * 10) - z;
             }
         }
+        seriesOptions.depth = seriesOptions.depth || 25;
```

Comment:
I suggest to set 25 as value in defaultOptions instead of hardcoding.

Reply:
We had similar discussion with @pawelfus about the above code and unfortunately due to a fact that we are not creating new series type but just extend existing one, we should keep the old way of this declaration (this is the same code as before but just moved into another place). However, I am totally with you about this line of code, had my own doubts about it and I think in future it should be a separate series type (column-3d) where you can add this option. This will also solve typescript problems.